docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -0,0 +1,220 @@
+local g = import 'grafana-builder/grafana.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'node-cluster-rsrc-use.json':
+      local legendLink = '%s/dashboard/file/k8s-node-rsrc-use.json' % $._config.grafana_prefix;
+
+      g.dashboard('USE Method / Cluster')
+      .addRow(
+        g.row('CPU')
+        .addPanel(
+          g.panel('CPU Utilisation') +
+          g.queryPanel(|||
+            (
+              instance:node_cpu_utilisation:rate1m{%(nodeExporterSelector)s}
+            *
+              instance:node_num_cpu:sum{%(nodeExporterSelector)s}
+            / ignoring (instance) group_left
+              sum without (instance) (instance:node_num_cpu:sum{%(nodeExporterSelector)s})
+            )
+          ||| % $._config, '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          // TODO: Is this a useful panel?
+          g.panel('CPU Saturation (load1 per CPU)') +
+          g.queryPanel(|||
+            (
+              instance:node_load1_per_cpu:ratio{%(nodeExporterSelector)s}
+            / ignoring (instance) group_left
```

Comment:
Shouldn't the instance label be the same on both sides?

Reply:
The right side doesn't have an `instance` label. It's counting `without (instance)` to normalize the value for the stacked graph in this panel.